test/functional/feature_block.py

```diff
@@ -95,16 +100,21 @@ def run_test(self):
         self.save_spendable_output()
         self.sync_blocks([b0])
 
+        # These constants chosen specifically to trigger an immature coinbase spend
+        # at a certain time below.
+        NUM_BUFFER_BLOCKS_TO_GENERATE = 100
```

Comment:
Why are both of these increasing by one?

Reply:
Just because I added an additional coinbase to (try to) spend for the new tests included below.

Reply:
Yep, good point. Fixed.